### PR TITLE
Automatically set the speaker on when turning on video

### PR DESCRIPTION
**Pull request by JPonte**

### CODE REVIEW

File: app/src/main/scala/com/waz/zclient/calling/controllers/CallController.scala [hunk 1]
```diff
@@ -204,6 +204,7 @@ class CallController(implicit inj: Injector, cxt: WireContext, eventContext: Eve
     verbose(s"toggleVideo")
     updateCall { case (call, cs) =>
       import VideoState._
+      if (call.videoSendState == Stopped) callingZms.head.flatMap(_.mediamanager.setSpeaker(true))
```
Comment (deanrobertcook) [inline app/src/main/scala/com/waz/zclient/calling/controllers/CallController.scala]: Maybe we should be doing this in SE?

Comment (JPonte) [reply app/src/main/scala/com/waz/zclient/calling/controllers/CallController.scala]: Maybe. Not sure if we want every `setVideoSendState(Started)` to also turn on the speaker or just the calls from this button.

Comment (deanrobertcook) [reply app/src/main/scala/com/waz/zclient/calling/controllers/CallController.scala]: well yeah - if the video is on, then so the speaker should be too, right?

**Comment by deanrobertcook** [comment]
logic moved to SE as discussed

**Pull request closed without merging**